db/db_basic_test.cc

```diff
@@ -1822,7 +1838,14 @@ INSTANTIATE_TEST_CASE_P(
                       std::make_tuple(false, true, false, true),
                       std::make_tuple(false, true, true, false),
                       std::make_tuple(true, true, true, false),
-                      std::make_tuple(false, true, false, false)));
+                      std::make_tuple(false, true, false, false),
```

Comment:
Any reason we're testing only 14 out of the 16 possible combinations here? If all 16 are supported, I think we could consider using `Combine(Bool(), Bool(), Bool(), Bool())`.